app/main.js

```diff
@@ -573,6 +574,17 @@ function getTrayMenu () {
           click: function () {
             pauseBreaks(3600 * 5 * 1000)
           }
+        }, {
+          label: i18next.t('main.untilMorning'),
+          click: function () {
+            const morningTimes = loadMorningTime()
+            if (morningTimes === null) {
+              console.log('Failed to load morning time, cannot pause')
+              return
+            }
+            const untilMorning = Utils.millisUntil(...morningTimes)
+            pauseBreaks(untilMorning)
```

Comment:
can you create a class for this in `/utils`? so in the end we would only call `pauseBreaks(Something.untilMorning())`or so

main.js is way too big already

Reply:
I'd suggest to make it just a file with a few functions that will return morning time, there is no need in a class imo.